src/main/java/it/gov/pagopa/pu/workflow/wf/pagopa/send/wfsendnotification/SendNotificationProcessWFImpl.java

```diff
@@ -25,6 +26,7 @@
 public class SendNotificationProcessWFImpl implements SendNotificationProcessWF, ApplicationContextAware {
   public static final String TASK_QUEUE_SEND_NOTIFICATION_PROCESS = "SendNotificationProcessWF";
   public static final String TASK_QUEUE_SEND_NOTIFICATION_PROCESS_LOCAL_ACTIVITY = "SendNotificationProcessWF_LOCAL";
+  public static final String TASK_QUEUE_SEND_NOTIFICATION_DATE_RETRIEVE_LOCAL_ACTIVITY = "SendNotificationDateRetrieveWF_LOCAL";
```

Comment:
we could use TASK_QUEUE_SEND_NOTIFICATION_PROCESS_LOCAL_ACTIVITY, instead of configuring a new one taskQueue